pkg/controller/kibana/init_configuration.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	kbv1 "github.com/elastic/cloud-on-k8s/pkg/apis/kibana/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	InitConfigContainerName = "elastic-internal-init-config"
+
+	// InitConfigScript is a small bash script to prepare the Kibana configuration directory
+	InitConfigScript = `#!/usr/bin/env bash
+set -eux
+
+init_config_initialized_flag=/usr/share/kibana/config/elastic-internal-init-config.ok
```

Comment:
is it only to avoid the cost of re-doing `ln -sf`?

Reply:
This is to avoid the content of the script to be run twice, I agree it should be safe to remove it here. But I also want to avoid `Setup Kibana configuration` to be printed several times in the logs. It's harmless indeed, but I think it can be confusing for the user/support to see those messages while the container is already running for a while.